doc/api/fs.md

```diff
@@ -2529,6 +2529,19 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the old path name of a file and rename it to the path
+name provided in the second parameter.
+
+Here is an example below (assuming the file "firstfile.txt" already exists):
+
+```js
+fs.rename('firstfile.txt', 'secondfile.txt', (err) => {
+	if(err) throw err;
+	console.log("Rename complete!");
```

Comment:
Please don't use tabs for indentation, two spaces are appropriate. Also, use single quotes whenever possible (`'Rename complete!'`).